JASP-Engine/JASP/R/bayesianmetaanalysis.R

```diff
@@ -1655,12 +1650,12 @@ BayesianMetaAnalysis <- function(jaspResults, dataset, options) {
                                                  pizzaTxt = pizzaTxt,
                                                  hasRightAxis = TRUE,
                                                  yName = yName,
-                                                 addEvidenceArrowText = FALSE,
-                                                 evidenceLeveltxt = FALSE
-                                                 )
+                                                 evidenceTxt  = paste('Evidence for r', H[1], ':'),
+                                                 arrowLabel  = c("Evidence~'for f'~H[1]", "Evidence~'for r'~H[1]")
```

Comment:
Would be good to use the gettext construction (at least in "Evidence for")

Reply:
Ah yes, of course!